build/test-page/index.html

```diff
@@ -6,10 +6,10 @@
 
         <script type = 'text/javascript' src = 'https://d3js.org/d3.v3.min.js'></script>
         <script type = 'text/javascript' src = 'https://rawgit.com/SheetJS/js-xlsx/master/dist/xlsx.full.min.js'></script>
-        <script type = 'text/javascript' src = 'https://rawgit.com/RhoInc/Webcharts/master/build/webcharts.js'></script>
```

Comment:
Don't think I'd actually change this link here since the branch is only temporary. Maybe set up a gist to demo and let users know that they can use the updated version or webcharts for better performance.